querybook/server/lib/query_review/all_query_review_handlers.py

```diff
@@ -0,0 +1,12 @@
+from lib.utils.import_helper import import_module_with_default
+from lib.query_review.base_query_review_handler import BaseQueryReviewHandler
+
+ALL_PLUGIN_QUERY_REVIEW_HANDLERS = import_module_with_default(
+    "query_review_handler_plugin",
+    "ALL_PLUGIN_QUERY_REVIEW_HANDLERS",
+    default=[BaseQueryReviewHandler()],
+)
+
+
+def get_query_review_handler() -> BaseQueryReviewHandler:
+    return ALL_PLUGIN_QUERY_REVIEW_HANDLERS[0]
```

Comment:
I think we dont really need multiple handlers configured. If multiple handler is needed, then it should be able to be configured per engine?

Also, I think it doesn't need to provide the base handler as default, as it seems not meaningful to use it alone without custom approve/reject logic.